alert-database/src/main/java/com/synopsys/integration/alert/database/notification/NotificationEntity.java

```diff
@@ -121,6 +125,14 @@ public OffsetDateTime getProviderCreationTime() {
         return providerCreationTime;
     }
 
+    public boolean getProcessed() {
+        return processed;
+    }
+
+    public void setProcessed() {
```

Comment:
This method name confused me in another file, we should change the name as setProcessed implies a simple setter where as we are explicitly setting this to true. Something as simple as setProcessedToTrue() would be easier to understand when reading the method elsewhere.